_includes/hero.html

```diff
@@ -1,11 +1,102 @@
+        border-radius: 50%;
+        animation: bounce 2s infinite;
+    }
+
+    /* Add custom CSS to style the arrow */
+    .arrow-down {
+        position: absolute;
+        bottom: calc(2.5% + 72px);
+        right: 8%;
+        font-size: 24px;
+        color: #fff; /* Change color if needed */
+        animation: bounce 2s infinite;
+        cursor: pointer;
+    }
+
+    /* Bounce animation same as before */
+    @keyframes bounce {
+        0%, 20%, 50%, 80%, 100% {
+            transform: translateY(0);
+        }
+        40% {
+            transform: translateY(-20px);
+        }
+        60% {
+            transform: translateY(-10px);
+        }
+    }
+
+    /* Add custom CSS for the "Scroll Down!" text */
+    .scroll-down-text {
+        position: absolute;
+        bottom: calc(5% + 30px);
+        right: 5.3%;
+        color: #fff;
+        font-size: 18px;
+    }
+
+    /* Add custom CSS for the hero section wrapper */
+    .hero-wrapper {
+        height: calc(100vh - 52px);
+        overflow: hidden;
+    }
+</style>
+
+{% if page.hero_image %}
+<div class="hero-wrapper">
+{% endif %}
+
+    <section class="hero {% if page.hero_height %} {{ page.hero_height }} {% else %} is-medium {% endif %} is-bold is-primary" {% if page.hero_image %} style="background: url('{{ page.hero_image }}') no-repeat center center; background-size: cover; height: 100%;" {% endif %}>
+        <div class="hero-body {% if page.hero_darken %} hero-darken {% endif %}">
+            <div class="container">
+                <h1 class="title is-2">{{ page.title }}</h1>
+                <p class="subtitle is-3">{{ page.subtitle }}</p>
+                {% if page.hero_link %}
                 <a href="{{ page.hero_link | relative_url }}" class="button is-info is-large">{{ page.hero_link_text }}</a>
-            {% endif %}
+                {% endif %}
+            </div>
         </div>
-    </div>
-</section>
\ No newline at end of file
+
+        {% if page.hero_image %}
+        <div class="arrow-container">
+            <!-- Use Font Awesome arrow icon -->
+            <i class="fa fa-arrow-down arrow-down" aria-hidden="true" onclick="scrollToNextSection()"></i>
+            <!-- Add the "Scroll Down!" text -->
+            <div class="scroll-down-text">Scroll Down!</div>
+        </div>
+        {% endif %}
+    </section>
+{% if page.hero_image %}
+    <!-- Add a target section that you want to scroll to -->
+    <div id="target-section" style="height: 100vh; background-color: #f5f5f5;"></div>
+</div>
+{% endif %}
+
+<script>
+    {% if page.hero_image %}
+        function scrollToNextSection() {
+            const targetSection = document.getElementById("target-section");
+            const scrollOffset = targetSection.getBoundingClientRect().top + window.scrollY;
+            window.scrollTo({ top: scrollOffset, behavior: "smooth" });
+        }
+
+
+        document.querySelector('.arrow-down').addEventListener('click', scrollToNextSection);
```

Comment:
Instead of wrapping all of this in a condition on hero_image, can you be more restrictive in the query selector so that it only attaches the event listener in the right condition?

Reply:
https://github.com/FRCTeam1987/teambroncobots.com/commit/5a553cf06c247558777edc0fe740c1cec0e9c854